src/batou_ext/modsecurity.py

```diff
@@ -0,0 +1,141 @@
+from textwrap import dedent
+import batou.component
+import batou.lib.file
+import batou_ext.nix
+
+
+@batou_ext.nix.rebuild
+class Modsecurity(batou.component.Component):
+    '''
+    Performing initial setup of modsecurity for nginx with the OWASP Core Rule Set
+    configuration framework. Gives an interface of blocking and Paranoia Level.
+
+    Usage:
+    The component is depending on nginx and expects /etc/local/nginx/modsecurity
```

Comment:
The component expects that a managed nginx installation based on the Flying Circus platform is enabled (e.g. by enabling the Webgateway role).